ui/src/facepile.tsx

```diff
@@ -16,18 +16,20 @@ export interface Facepile {
   name?: Id
   /** Maximum number of personas to be displayed. */
   max?: U
+  /** A value to be submitted when the add button is clicked. If value is not provided, true will be submitted*/
+  value?: S
 }
 
 export const XFacepile = ({ model }: { model: Facepile }) => {
   const
-    { name, max, items } = model,
+    { name, max, items, value } = model,
     onClick = () => {
       if (name) {
         if (name.startsWith('#')) {
           window.location.hash = name.substr(1)
           return
         }
-        wave.args[name] = true
+        wave.args[name] = value || true
```

Comment:
This will fail if `value` is specified as an empty string. Better go with
```suggestion
        wave.args[name] = value === undefined || value
```

Reply:
I don't see the point of letting the user send an empty string but If that's the case I will change.